index.html

```diff
@@ -12,9 +12,12 @@
       <input type="text" id="color-field"></input>
       <button id="set-color">Set Color</button>
     </form>
-    <div class="brush"></div>
+    <div class="brush" id='box1'></div>
+    <div class="brush" id='box2'></div>
+    <div class="brush" id='box3'></div>
+    <div class="brush" id='box4'></div>
   </div>
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+  <script src="/Users/timothygregg/wdi/exercises/jquery-inclass-blawg/js/jquery-2.1.4.js"></script>
```

Comment:
this will only load jquery if on your computer. remove everything before `/jquery-2.1.4.js`